boards/arc/em_starterkit/em_starterkit_defconfig

```diff
@@ -1,5 +1,5 @@
 CONFIG_ARC=y
-CONFIG_SOC_EM9D=y
+CONFIG_SOC_EM7D=y
```

Comment:
there is already a em_starterkit_em7d_defconfig for the em7d, can you use that instead of em7d? what happens with the em9d? you do not want it configured as a board at all?

Reply:
It's caused by my misunderstanding of new cmake build system. I will make a fix to all defconfig， update to em_starterkit version 2.3.